### ISSUE TITLE
OTLP/GRPC exporter implementation - `Cannot convert '' to string` exception on empty `parent_span_id` with ext-protobuf

### ISSUE THREAD

**Issue by parshikov** [open-telemetry/opentelemetry-php#357]
**Environment** 
docker image: `php:7.4.20-fpm-alpine`
related php-ext enabled: `grpc-1.38.0`, `protobuf-3.17.3`
library: `open-telemetry/opentelemetry: 0.0.2`,
jaeger metrics collector: `jaegertracing/opentelemetry-all-in-one`

**Steps to reproduce**
```php

use OpenTelemetry\Contrib\OtlpGrpc\Exporter;
use OpenTelemetry\Sdk\Resource\ResourceInfo;
use OpenTelemetry\Sdk\Trace\Clock;
use OpenTelemetry\Sdk\Trace\Sampler\AlwaysOnSampler;
use OpenTelemetry\Sdk\Trace\SpanProcessor\BatchSpanProcessor;
use OpenTelemetry\Sdk\Trace\TracerProvider;
use OpenTelemetry\Trace\SpanKind;
use OpenTelemetry\Trace\SpanStatus;

$exporter = new Exporter('home.docker.internal:55680'); // point to jaegertracing/opentelemetry-all-in-one

$tracer =  (new TracerProvider(
    ResourceInfo::defaultResource(),
    new AlwaysOnSampler())
)->addSpanProcessor(new BatchSpanProcessor($exporter, Clock::get()))->getTracer('tracername', 'doesntmatter');

$span = $tracer->startAndActivateSpan($request->getMethod(), SpanKind::KIND_SERVER);

if ($span->isRecording()) {
    $span->setAttribute('attribute1', 'value1');
}

$span->setSpanStatus(SpanStatus::OK);
$tracer->endActiveSpan();

// To easily catch exception make `\OpenTelemetry\Sdk\Trace\Tracer::$provider` public
$tracer->provider->shutdown();
```

**What is the expected behavior?**
Tracing information appears in jaeger

**What is the actual behavior?**
Exception: **Cannot convert '' to string**
```
Google\Protobuf\Internal\Message::__construct
vendor/open-telemetry/opentelemetry/proto/Opentelemetry/Proto/Trace/V1/Span.php:244
```

**Additional context**
This exception does not appear in next cases (independently of each other):

- Using `google/protobuf` instead of `ext-protobuf`
- Using non-empty `parentId` in context as well as using invalid `parentId` as parent one 
```php
- 'parent_span_id' => $parent_span_id ? hex2bin($parent_span_id) : null,
+ 'parent_span_id' => $parent_span_id ? hex2bin($parent_span_id) : hex2bin(\OpenTelemetry\Sdk\Trace\SpanContext::INVALID_SPAN),
```
during row initialisation in `vendor/open-telemetry/opentelemetry/contrib/OtlpGrpc/SpanConverter.php:91`.

**NB!** This issue can be related to the protobuf extension, not opentelemetry package

**Comment by bobstrecansky** [comment]
@parshikov - I think this should be solved with #352 - can you please reopen if it's not completely fixed for you?

**Comment by brettmc** [comment]
This is still an issue with the `ext-protobuf` extension (3.19.1 and 3.19.2), but not with the protobuf package. This has implications for both of the OTLP exporters, which may use the protobuf extension for improved performance.
